client/state/selectors/get-blog-stickers.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns the blog stickers for a certain site.
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {Number}  blogId  The ID of the site we're querying
+ * @return {Array} Blog stickers
+ */
+export default function getBlogStickers( state, blogId ) {
+	return get( state.sites.blogStickers.items, [ blogId ], null );
```

Comment:
Actually, bringing back the `get`, because it'll return `undefined` rather than `null` if we don't have stickers for a blog.